openpype/modules/kitsu/plugins/publish/collect_kitsu_entities.py

```diff
@@ -29,7 +29,7 @@ def process(self, context):
             if not zou_asset_data:
                 raise ValueError("Zou asset data not found in OpenPype!")
 
-            task_name = instance.data.get("task")
+            task_name = instance.data.get("task", context.data.get("task"))
```

Comment:
We would like to get rid of using context task in these situations. Instance should know it's task. Was there a use-case when it was not filled?

Reply:
`Fixes an issue with Photoshop's review instance not having task in data.`

Reply:
Yes - it was in Photoshop [Collect Review which creates an instance during publishing](https://github.com/ynput/OpenPype/blob/develop/openpype/hosts/photoshop/plugins/publish/collect_review.py#L42-L52).

I've mentioned on Discord that it should've likely been refactored to a new style AutoCreator because before the refactor to new publisher I assume one would've been able to disable the review instance in the publisher. Looking at the code now I suspect that isn't possible.

(I haven't run Photoshop in OP myself)